README.rst

```diff
@@ -682,19 +682,28 @@ When training and deploying training scripts, SageMaker runs your Python script
 
 SageMaker runs MXNet Estimator scripts in either Python 2.7 or Python 3.5. You can select the Python version by passing a ``py_version`` keyword arg to the MXNet Estimator constructor. Setting this to ``py2`` (the default) will cause your training script to be run on Python 2.7. Setting this to ``py3`` will cause your training script to be run on Python 3.5. This Python version applies to both the Training Job, created by fit, and the Endpoint, created by deploy.
 
-Your MXNet training script will be run on version 0.12 of MXNet, built for either GPU or CPU use. The decision to use the GPU or CPU version of MXNet is made by the train_instance_type, set on the MXNet constructor. If you choose a GPU instance type, your training job will be run on a GPU version of MXNet. If you choose a CPU instance type, your training job will be run on a CPU version of MXNet. Similarly, when you call deploy, specifying a GPU or CPU deploy_instance_type, will control which MXNet build your Endpoint runs.
-
-Each Docker container has the following dependencies installed:
+Your MXNet training script will be run on version 1.0.0 (by default) or 0.12 of MXNet, built for either GPU or CPU use. The decision to use the GPU or CPU version of MXNet is made by the train_instance_type, set on the MXNet constructor. If you choose a GPU instance type, your training job will be run on a GPU version of MXNet. If you choose a CPU instance type, your training job will be run on a CPU version of MXNet. Similarly, when you call deploy, specifying a GPU or CPU deploy_instance_type, will control which MXNet build your Endpoint runs.
 
+The MXNet Docker container version 1.0 has the following dependencies installed:
 -  Python 2.7 or Python 3.5, depending on the ``py_version`` argument on
    the MXNet constructor.
 -  MXNet 0.12, built for either GPU or CPU, depending on the instance
    type for training or deploying.
 -  CUDA 9.0
 -  numpy 1.12
 
+The MXNet Docker container version 1.1 has the following dependencies installed:
+-  Python 2.7 or Python 3.5, depending on the ``py_version`` argument on
+   the MXNet constructor.
+-  MXNet 1.0.0, built for either GPU or CPU, depending on the instance
+   type for training or deploying.
+-  CUDA 9.0
+-  numpy 1.13.3
```

Comment:
this seems like it'll become unwieldy as new versions are added, especially since half the list is the same for each (1.0 vs 1.1). Is there a way to maybe put this into some kind of table-like display? (open to other suggestions, too, of course)

same thought below for TF